SharedPackages/BrowserServicesKit/Sources/Subscription/Managers/SubscriptionManagerV2.swift

```diff
@@ -280,6 +280,7 @@ public final class DefaultSubscriptionManagerV2: SubscriptionManagerV2 {
     public func loadInitialData() async {
         Logger.subscription.log("Loading initial data...")
         do {
+            _ = try? await getTokenContainer(policy: .localValid)
```

Comment:
this is needed for fixing the migration, broken by this PR changes

Reply:
Removing one of the 2 notifications changed something in the migration process, without that the subscription needs time to appear in the settings and, with bad networking, could look like the subscription is not there, this is performing the subscription fetch before terminating the migration